helpers/config.py

```diff
@@ -312,16 +350,29 @@ def __create_directory(self):
         while True:
             kobodocker_path = CLI.colored_input("", CLI.COLOR_SUCCESS,
                                                 self.__config.get("kobodocker_path"))
-            if os.path.isdir(kobodocker_path):
-                break
-            else:
-                try:
-                    os.makedirs(kobodocker_path)
-                    break
-                except Exception as e:
-                    CLI.colored_print("Could not create directory {}!".format(kobodocker_path), CLI.COLOR_ERROR)
-                    CLI.colored_print("Please make sure you have permissions and path is correct", CLI.COLOR_ERROR)
 
+            if kobodocker_path.startswith("."):
+                base_dir = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
+                kobodocker_path = "{}/{}".format(base_dir, kobodocker_path)
+
+            CLI.colored_print("Please confirm path [{}]".format(kobodocker_path),
+                              CLI.COLOR_SUCCESS)
+            CLI.colored_print("\t1) Yes")
+            CLI.colored_print("\t2) No")
+
+            if CLI.get_response([Config.TRUE, Config.FALSE], Config.TRUE) == Config.TRUE:
+
+                if os.path.isdir(kobodocker_path):
+                    break
+                else:
+                    try:
+                        os.makedirs(kobodocker_path)
+                        break
+                    except Exception as e:
+                        CLI.colored_print("Could not create directory {}!".format(kobodocker_path), CLI.COLOR_ERROR)
+                        CLI.colored_print("Please make sure you have permissions and path is correct", CLI.COLOR_ERROR)
```

Comment:
Should execution be allowed to continue if directory creation fails?

Reply:
nope! Fixed that.